src/main/java/com/epam/lab/payments/domain/CreditCardEntity.java

```diff
@@ -38,4 +38,8 @@
     @ManyToOne
     @JoinColumn(name = "account_id", referencedColumnName = "id", nullable = false)
     private BankAccountEntity bankAccountByAccountId;
```

Comment:
Are you sure it is a good name for the entity field?
The same question for `userByUserId`

Reply:
rework to bankAccount and user